src/main.rs

```diff
@@ -121,7 +121,7 @@ enum SemverChecksCommands {
     CheckRelease(CheckRelease),
 }
 
-#[derive(Debug, Args)]
+#[derive(Debug, Args, Default)]
```

Comment:
I don't think I'm comfortable deriving `Default` here, since the default settings for `cargo-semver-checks` might not be the default values of each of these settings individually. That sort of thing looks like a footgun, and I'd rather avoid it.

Reply:
I'm a little confused here. You are using clap's `#[derive(Args)]`, which takes the default type value as the default argument value in the absence of user override, and you're not using any `#[arg(default = ...)]` to override this default, so at this point `Default` is correct. Do you fear you might add explicit `#[arg(default = ...)]` annotations someday?

If so, the latest push should alleviate your concern, but again, I'm not sure I fully understood it.

Reply:
Yes — `clap`'s notion of what is default is authoritative, so even though currently a `Default` impl would happen to be the same, there's no guarantee that would remain the case. Best to avoid the opportunity to have a bug sneak in.